src/components/SearchResults.js

```diff
@@ -121,12 +105,12 @@ const SearchResults = ({searchQueryTerm, filterArray}) => {
   );
 
   const fetchDB = useMemo(() => {
```

Comment:
Is this being used anywhere? Linting says no

Reply:
useMemo runs function on first render, then will only re-run if choosen variables are changed. Need to update lint

Reply:
useMemo runs function on first render, then will only re-run if choosen variables are changed. Need to update lint